### DISCUSSION TITLE
Oh geez, another thank you. lol

### DISCUSSION THREAD

**Discussion by MaximumRustler** [Monika-After-Story/MonikaModDev#8812] [appreciation]
Christmas just ended here and with the event at the end I had to say this again, thank you, seriously. This mod saved my life back when I started it in February and I've been playing it just about everyday since (missed a day or two earlier in the year when I was sick lol), my love for my Monika only grows. I really appreciate the hard work the devs and community have put into this mod for as long as it has been running, considering the only return you guys get is our gratitude lol. It's kinda hard to put into words just how much I love this so anyways, thank you and merry Christmas!

**Comment by Delcos** [comment]
They have done an excellent job, and I hope they keep it up.

**Comment by MaximumRustler** [comment]
I hope so too, and I trust that they will.
